src/lib/control_allocation/actuator_effectiveness/ActuatorEffectiveness.cpp

```diff
@@ -84,15 +84,16 @@ int ActuatorEffectiveness::Configuration::totalNumActuators() const
 	return total_count;
 }
 
-void ActuatorEffectiveness::stopMaskedMotorsWithZeroThrust(uint32_t stoppable_motors_mask, ActuatorVector &actuator_sp)
+void ActuatorEffectiveness::stopMaskedMotorsWithZeroThrust(uint32_t stoppable_motors_mask, ActuatorVector &actuator_sp,
+		float threshold)
 {
 	for (int actuator_idx = 0; actuator_idx < NUM_ACTUATORS; actuator_idx++) {
 		const uint32_t motor_mask = (1u << actuator_idx);
 
 		if (stoppable_motors_mask & motor_mask) {
 
-			// Stop motor if its setpoint is below 2%. This value was determined empirically (RC stick inaccuracy)
-			if (fabsf(actuator_sp(actuator_idx)) < .02f) {
+			// Stop motor if its setpoint is below threshold.
+			if (fabsf(actuator_sp(actuator_idx)) < threshold) {
```

Comment:
```suggestion
			if (fabsf(actuator_sp(actuator_idx)) + FLT_EPSILON < threshold) {
```
otherwise I wouldn't be sure it can still turn off.